solidity/contracts/BondedECDSAKeep.sol

```diff
@@ -485,10 +485,14 @@ contract BondedECDSAKeep is IBondedECDSAKeep {
 
         memberETHBalances[_member] = 0;
 
+        address beneficiary = tokenStaking.beneficiaryOf(_member);
+        require(
+            beneficiary != address(0),
+            "Beneficiary not defined for the operator"
+        );
```

Comment:
Do we need this check here? I think it's not possible to happen. To become a keep member, stake delegation has to exist and the beneficiary has to be defined. If we want to guard against `0x0` as a beneficiary, it should be done either in the dApp or in the staking contract.

Reply:
3d2c4513add72429288c48f48b59527aad246fb1